config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml

```diff
@@ -65,3 +65,28 @@ presubmits:
         - prow/cluster
     annotations:
       testgrid-create-test-group: 'true'
+  - name: pull-test-infra-prow-checkconfig
+    branches:
+      - master
+    decorate: true
+    cluster: build
```

Comment:
this cluster doesn't exist

new presubmits should be `always_run: false` or `optional: true` until they are vetted